messageService/message_test.go

```diff
@@ -0,0 +1,29 @@
+package messageService
+
+import (
+	"testing"
+	"time"
+	"fmt"
+)
+
+var (
+	Type_ = "messageService"
+	Command = "send message"
+	Sender = 1
+	Receiver = 2
+	CurrentTime = time.Now
+	Auth = "token"
+	Text = "msg from user 1 to user 2"
+	)
+
+func TestMarshalMessage(t *testing.T) {
```

Comment:
Done. Tests for all functions for message service have been developed and checked.